.azure-pipelines/azure-pipelines-Official.yml

```diff
@@ -0,0 +1,29 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger: none
+
+schedules:
+- cron: "0 0 * * *"
```

Comment:
>cron: "0 0 * * *" [](start = 2, length = 17)

Is it only trigged by time or time + new commit?

Reply:
Only daily build, you mean we do not need to build if not change, right?

Reply:
I mean building at least daily, and trigger if no jobs and new commits coming.

---
In reply to: [580004156](https://github.com/Azure/sonic-buildimage/pull/6837#discussion_r580004156) [](ancestors = 580004156)

Reply:
Changed to daily build.